### ISSUE TITLE
[google_maps_flutter_web] Maps view detects gesture even when a widget is over it

### ISSUE THREAD

**Issue by tapizquent** [flutter/flutter#74217]
## Steps to Reproduce

1. Run `flutter create bug`.
2. Update the files as follows: 
    Add to current page. Set up with correct values for it to display properly.
3. Push a new route.
4. Try to click on anything in that route.
The map is registering gestures and is moving, even though it's not visible but behind the newly pushed route.

**Expected results:** 
The maps should not continue registering hit tests when its not visible or is being blocked by another widget.

**Actual results:** 
The maps detects gestures and conflict with newly pushed route.

<details>
  <summary>Logs</summary>

```
```

```
Just some TODOs
```

```
[✓] Flutter (Channel beta, 1.25.0-8.3.pre, on macOS 11.1 20C69 darwin-x64, locale en-US)
    • Flutter version 1.25.0-8.3.pre at /Users/josetapizquent/Projects/Tools/flutter
    • Framework revision 5d36f2e7f5 (4 days ago), 2021-01-14 15:57:49 -0800
    • Engine revision 7a8f8ca02c
    • Dart version 2.12.0 (build 2.12.0-133.7.beta)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/josetapizquent/Library/Android/sdk
    • Platform android-30, build-tools 29.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.52.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.18.1

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 87.0.4280.141
    ! Error: TapPhone is not connected. Xcode will continue when TapPhone is connected. (code -13)

• No issues found!
```

</details>

**Comment by darshankawar** [comment]
@tapizquent 
Can you check out [this](https://github.com/flutter/flutter/issues/73830) issue and see if it resemble yours and if the solution mentioned in it works for you ?

If not, please provide a complete minimal reproducible code sample that shows the issue.
Thanks.

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
